api/python/cell_census/pyproject.toml

```diff
@@ -31,7 +31,7 @@ dependencies= [
     "numpy>=1.21",
     "requests",
     "tiledb>=0.19.0",
-    "tiledbsoma==0.5.0a3",
+    "tiledbsoma==0.5.0a4",
```

Comment:
at this point, 0.5.0a5 has been released.  update or separate PR?  I'm fine with either.

Reply:
Intended to address a5 in a new issue/PR.  With the now quicker tiledbsoma alpha release cadence, I figured this PR might be chasing the latest version indefinitely.

Reply:
https://github.com/chanzuckerberg/cell-census/issues/81